main.cpp

```diff
@@ -136,25 +195,25 @@ int main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    auto circleArea = calcCircleArea(2.5f);
     //2)
-    
+    createPerson(35,68.4f, 180.54f);
     //3)
-    
+    auto randInt = generateRandomInt();
     //4)
-    
+    auto studentGPA = calcGPA(88.3f, 0.65f, 94.2f)
     //5)
-    
+    auto matched = charMatch('1', '2');
     //6)
-    
+    auto castChar = charToInt('3');
     //7)
-    
+    auto isEqual = checkForEquality(7,6);
     //8)
-    
+    auto playerRating = calculatePlayerRating(15.6, 5.6, 4.8);
     //9)
-    
+    addChar(16,'6');
     //10)
-    
+    auto newPitch = addOctave(52);
     
     ignoreUnused(carRented);
     std::cout << "good to go!" << std::endl;

```

Comment:
```
9) in the main function at the end: 
    for each of those functions declared, 
        a) write out how the function would look if called with correct arguments
        b) if the function returned anything, store it in a local variable via the 'auto' keyword.
        c) pass the local variables to ignoreUnused() as you did in 2b)
    see main() for an example of this.
```
- [ ] revise

Reply:
Added the variables to the ignoreUnused call in main() and fixed calcGPA call with closing semicolon